.travis.yml

```diff
@@ -50,7 +50,7 @@ after_success:
         - test 1 = "$SKIP_COV" || ./rebar3 codecov analyze
         - test 1 = "$SKIP_COV" || codecov --disable=gcov --env PRESET
         - test 1 = "$SKIP_COV" || ./rebar3 coveralls send
-        - if [ -n "${DOCKERHUB_PASS}" ] && [ $PRESET = 'internal_mnesia' ] && [ $TRAVIS_OTP_RELEASE = "19.3" ];
+        - if [ -n "${DOCKERHUB_PASS}" ] && [ $PRESET = 'internal_mnesia' ] && [ $TRAVIS_OTP_RELEASE = "20.3" ];
```

Comment:
Given that SSL in 20.3 requires special treatment, I think we should push an image built with 21.3 to Dockerhub.

Reply:
do you want to run `internal_mnesia` on OTP 21.3?

Reply:
Not necessarily. We may simply change the job responsible for pushing to Docker to e.g. `mysql_redis` and have it use `with-all` option (and have `internal_mnesia` use `with-none` to keep the build time balance).

Reply:
than i think `elasticsearch_and_cassandra_mnesia` is the best candidate, since it runs only limited `mam.spec`. but it's also OTP 20.3